nestkernel/mpi_manager.cpp

```diff
@@ -120,9 +120,19 @@ nest::MPIManager::init_mpi( int* argc, char** argv[] )
     set_communicator( MPI_COMM_WORLD );
 #endif /* #ifdef HAVE_MUSIC */
   }
```

Comment:
On line 116 above is a C-style cast to `MPI_Comm`. Could you replace that with a proper `static_cast`? (Minor tidying up should be allowed in PRs ;)).